### DISCUSSION TITLE
My application cannot be reviewed until you fix the following

### DISCUSSION THREAD

**Discussion by VEBST3R** [community/community#88449] [GitHub Education] [answered]
### Hi everyone,

When I clicked the Process my application button I got the following message:

Your application cannot be reviewed until you fix the following:
Please correct your GitHub billing information with your last name exactly as it appears in your academic affiliation document. If you have only a single legal name,
enter it in both the first and last name fields.
You appear not to be near any campus location for the school you have selected. Your academic affiliation documentation must indicate virtual learning if you are remote. Your must configure your browser and operating system to allow location access. You may not use a VPN.
If you believe that our information about your school is incomplete or incorrect e.g. missing campus location(s), then please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket.
Please correct your GitHub billing information with your first name exactly as it appears in your academic affiliation document. Do not use a variation of your name or a nickname.

But the problem is that in GitHub billing information my first and last name are filled in correctly (I've already changed the places and tried everything), and how can I confirm if I'm on vacation and I'm in another city not near the university?
![зображення_2024-01-16_231026667](https://github.com/community/community/assets/119488524/c14c1435-bcd3-4040-8832-32c79a04dbac)

**Comment by mishmanners** [comment, marked as answer]
It appears you don't meet the requirements for the second dot point. I'd wait until you have finished your vacation and you can reapply when you're in the correct location.
